templates/dashboard/product/stock_locations/modal_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+<form action="{% url 'dashboard:product-stock-location-delete' location.pk %}" method="post" novalidate>
+    {% csrf_token %}
+    <div class="modal-content">
+        <h5>{% trans "Are you sure?" %}</h5>
+
+        <p>
+            {% blocktrans %}You are about to delete a stock location <strong>{{ location }}</strong>.{% endblocktrans %}
```

Comment:
We should warn about any stock records that are about to be removed as well. Just a count should do. Ideally we'd ask the user to repeat the location name before the button becomes enabled.